packages/hardhat-plugin/src/index.ts

```diff
@@ -173,9 +178,14 @@ task("deploy")
     }
   );
 
-task("visualize")
+ignitionScope
+  .task("visualize")
   .addFlag("quiet", "Disables logging output path to terminal")
```

Comment:
We could remove this. It doesn't seem to be providing a huge value add. It is inherited from a previous iteration.

Reply:
It was being used to suppress opening during test runs. I am going to leave it for now.